app/markets/HistorySection.tsx

```diff
@@ -0,0 +1,53 @@
+import {
+  Icon,
+  TabBody,
+  TabGroup,
+  TabHeader,
+  TabPanel,
+  TabStyled,
+} from "swapr-ui";
+
+export const HistorySection = () => {
+  return (
+    <div className="p-2">
+      <TabGroup
+        onChange={(index: number) =>
+          console.log("Changed selected tab to:", index)
+        }
```

Comment:
This onChange shouldn't be required, right? If it's not, it can be deleted.

Reply:
yes, it's optional